packages/core/.eslintrc.json

```diff
@@ -0,0 +1,68 @@
+{
+  "plugins": ["@typescript-eslint", "simple-import-sort", "unused-imports"],
+  "extends": [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/recommended",
+    // configures both eslint-plugin-prettier and eslint-config-prettier
+    "plugin:prettier/recommended"
+  ],
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "ecmaVersion": 2020,
+    "sourceType": "module",
+    "project": "tsconfig.json"
+  },
+  "env": {
+    "es6": true,
+    "browser": true,
+    "node": true,
+    "jest": true
+  },
+  "rules": {
+    "simple-import-sort/imports": 2,
+    "simple-import-sort/exports": 2,
+    "prettier/prettier": 2,
+    "unused-imports/no-unused-imports": 2,
+    "no-debugger": 2,
+    "no-alert": 2,
+    "no-await-in-loop": 0,
+    "no-prototype-builtins": 0,
+    "no-return-assign": ["error", "except-parens"],
+    "no-restricted-syntax": [
```

Comment:
what happened to `no-console`? :stuck_out_tongue:

Reply:
Adding it back :)